hooks/MTA.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright 2016 Signal K <info@signalk.org> and contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict'
+
+ const utils = require('@signalk/nmea0183-utilities')
+
+ /*
+  * MTA - Mean Temperature of Air
+  *
+  *        0   1 2
+  *        |   | |
+  * $--MTA,x.x,C*hh<CR><LF>
```

Comment:
Since the unit field is in the sentence I think we should add a check that the value is actually `C` and not something else.

Reply:
Done. Also adding similar checks to the other two sentences (MWD, VWT)